tasks/mysql_secure_installation.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: Install python-mysqldb for Ansible
+  package: pkg=python-mysqldb state=present
+
+- name: root password is present
+  mysql_user: name=root host={{ item }} password={{ mysql_root_password }} state=present
+  with_items:
+    - '::1'
+    - '127.0.0.1'
+    - 'localhost'
+
+- name: root has .my.cnf
+  template: src=my.cnf.j2 dest=/root/.my.cnf 
+            owner=root group=root mode=0600
+  tags: my_cnf
+
+# Can use only if ansible ver => 2.1
```

Comment:
We want to support Ansible 1.9 for now, too. So the command following is a good idea.